app/src/main/java/com/x8bit/bitwarden/ui/platform/components/toggle/BitwardenWideSwitch.kt

```diff
@@ -100,7 +102,8 @@ fun BitwardenWideSwitch(
 
         Switch(
             modifier = Modifier
-                .height(56.dp),
+                .height(56.dp)
+                .testTag("SwitchToggle"),
```

Comment:
This seems way to generic, most of the places we use the switch already have the test tags.

Reply:
I decided to add this generic ID at the component level due to a refactor we made on the automation side.
Now, each `Switch` element will contain its name label and toggle element, making most of our validations easie

Reply:
I see, what about the description?